vpc.tf

```diff
@@ -64,11 +64,9 @@ resource "aws_subnet" "region_az_1_public" {
   cidr_block              = local.region_az_1_public_subnet
   map_public_ip_on_launch = true
 
-  tags = merge(local.tags, map(
```

Comment:
Why did you use `zipmap()` here instead of the `{ "key" = "name" }` syntax that you used elsewhere?

Reply:
I was experimenting with it because I didn't initially realize the new map syntax allowed me to use interpolation syntax in keynames. It should be removed in the latest commit to this PR branch.